apps/wing/src/commands/test/test.test.ts

```diff
@@ -158,10 +155,10 @@ describe("output-file option", () => {
     expect(testName).toBe("test.test.w");
     expect(writeResultsSpy.mock.calls[0][2]).toBe(outputFile);
 
-    expect(writeFileSpy).toBeCalledTimes(2);
-    const [filePath, output] = writeFileSpy.mock.calls[1];
-    expect(filePath).toBe("out.json");
-    expect(JSON.parse(output as string)).toMatchObject(OUTPUT_FILE);
```

Comment:
This test was getting flakey because it's spying on all `writeFile` calls, including those made by simulator resources (not really related to the CLI logic), so I refactored it a bit